### PR TITLE
#43 Implement sprint rating & Recent feedback

### LINKED ISSUE #43: Implement sprint rating & Recent feedback
As a trainee, I should be able to see sprint ratings & Recent feedback. You can check this on the trainee dashboard when you logged in as a trainee

Image

### PR DESCRIPTION
**Pull request by pacifiquemboni**
PR Description
This PR Implement sprint rating & Recent feedback

Description of tasks that were expected to be completed
As a trainee, I should be able to see sprint ratings & Recent feedback. 
How has this been tested?
Clone repo
Checkout branch `ft-sprint-ratings`
Make sure that you set the environment variables referring to .env.example
login using trainee credentials
Run` npm install`, then `npm run dev` to check if the app is running successfully

WhatsApp Image 2024-10-08 at 16 57 50_e43f0bd6

Screenshot_20241008_165319

### CODE REVIEW

File: assets/images/arrow-left.png
Comment (fredshema) [inline assets/images/arrow-left.png:1]: Please use `Ionicons` for icons, we don't use file icons

**Review by fredshema — CHANGES_REQUESTED** [review]

**Comment by fredshema** [comment]
@pacifiquemboni check on figma on the design, your designs doesn't match that

**Review by niyibi250 — CHANGES_REQUESTED** [review]
check conflict and resolve them

**Comment by Mbabazi12** [comment]
@pacifiquemboni this can't be changed into dark mode

**Comment by pacifiquemboni** [comment]
> @pacifiquemboni this can't be changed into dark mode

check again @Mbabazi12

**Comment by Mbabazi12** [comment]
> > @pacifiquemboni this can't be changed into dark mode
> 
> check again @Mbabazi12

Oky i checked and it looks good on my side but could you add the dark mode screenshot for everyone to see it. thank you

**Review by Mbabazi12 — APPROVED** [review]

**Review by niyibi250 — APPROVED** [review]
look good

**Comment by ncutixavier** [comment]
@pacifiquemboni  there is right padding that need to be checked

**Comment by pacifiquemboni** [comment]
> @pacifiquemboni there is right padding that need to be checked
worked on it @ncutixavier

**Review by fredshema — APPROVED** [review]

**Review by munezeromicha — APPROVED** [review]

File: assets/images/mdi-eye.png
Comment (ncutixavier) [inline assets/images/mdi-eye.png:1]: Remove this if it is not being used anywhere

File: components/sprintratings/index.tsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+import EvilIcons from '@expo/vector-icons/EvilIcons';
+import Fontisto from '@expo/vector-icons/Fontisto';
+
+export default function TraineeRatings() {
+  const [inputValue, setInputValue] = useState(''); // For general input
+  const [sprintFilter, setSprintFilter] = useState(''); // For filtering by sprint
+  const [selectedPhase, setSelectedPhase] = useState('Phase I'); // Default phase selection
+  const [isOpenModel, setIsOpenModel] = useState(false);
+  const [selectedFeedback, setSelectedFeedback] = useState<any>(null); // Store feedback of the selected rating
+  const [userToken, setUserToken] = useState<string | null>(null);
```
Comment (ncutixavier) [inline components/sprintratings/index.tsx]: Nit sure if we really need these comments!

File: components/sprintratings/index.tsx
Comment (ncutixavier) [inline components/sprintratings/index.tsx:1]: Are you planning to have more files in `sprintratings`?

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**